tracer/src/Datadog.Tracer.Native/debugger_constants.h

```diff
@@ -36,6 +36,67 @@ const WSTRING skip_assemblies[]{WStr("mscorlib"),
                                 WStr("ISymWrapper"),
                                 WStr("Datadog.Trace"),
                                 WStr("MSBuild")};
+
+enum class DynamicInstrumentationErrorCode
+{
+    ALREADY_INSTRUMENTED = 0,
+    NOT_SUPPORTED_METHOD_PROBE = 1,
+    NOT_SUPPORTED_LINE_PROBE = 2,
+    UNKNOWN_FAILURE_METHOD_PROBE = 3,
+    UNKNOWN_FAILURE_LINE_PROBE = 4,
+    CTOR_AND_CCTOR_NOT_SUPPORTED = 5,
+    BYREF_LIKE_RETURN_NOT_SUPPORTED = 6,
+    PROFILER_ASSEMBLY_NOT_LOADED_YET = 7,
+    METHOD_IL_IMPORT_FAILURE = 8,
+    LOCALS_PARSE_FAILURE = 9,
+    ADD_DYNAMIC_INSTRUMENTATION_LOCALS_FAILURE = 10,
+    BYREF_LIKE_TYPE_NOT_SUPPORTED = 11,
+    GET_DEBUGGER_LOCALS_FAILURE = 12,
+    METHOD_IL_EXPORT_FAILURE = 13,
+    METHOD_IS_ASYNC_FAILURE = 14,
+    TASK_RETURN_TYPE_RETRIEVAL_FAILURE = 15
+};
+
+const WSTRING general_error_message = WStr("Failed to instrument the method.");
+
+inline WSTRING GetDynamicInstrumentationErrorMessage(DynamicInstrumentationErrorCode errorCode)
+{
+    return general_error_message + WStr("[Error Code:") + shared::ToWSTRING(static_cast<int>(errorCode)) + WStr("]");
+}
+
+const WSTRING invalid_probe_method_already_instrumented =
+    WStr("The Dynamic Insturmentation has failed to place the probe because the corresponding method was previosuley "
+         "instrumented by another product.");
+const WSTRING invalid_method_probe_probe_is_not_supported =
+    WStr("The method where the probe should have been placed is not supported for now.");
+const WSTRING invalid_line_probe_probe_is_not_supported =
+    WStr("The line where the probe should have been placed is not supported for now.");
```

Comment:
Is there anything more specific we can say here? Do we know what type of situations lead to this error?

Reply:
I'm currently doing some changes that will eliminate the generic explanation of the line probes, the generic explanation of the method probe is there mainly for completion although we should not land on it (I can not see clear code paths that may yield `E_NOTIMPL` for method probe instrumentation).

Reply:
Deleted the generic line probe explanation in 8f3b029d9f707341ec353f2c187f0cfadd81a081.